code_snippets/api-monitor-schedule-downtime.py

```diff
@@ -9,5 +8,15 @@
 
 initialize(**options)
 
+# Repeat for 3 hours (starting now) on every week day for 4 weeks.
```

Comment:
I'm going to swap to an `until_date` since I think that's easier to wrap your head around for an example.